Cargo.toml

```diff
@@ -19,9 +19,10 @@ categories = ["data-structures", "no-std"]
 
 [dependencies]
 serde = { version = "1.0", optional = true, features = ["derive"] }
+borsh = { version = "1.5.4", optional = true, features = ["derive"] } # using 1.5.4 as it is good regarding syn
```

Comment:
What do you mean 'good regarding syn'?

Reply:
made >=1.5.4, <1.6. 1.5.3 was weirdly broken for some combination of features.